src/library_sockfs.js

```diff
@@ -563,6 +563,9 @@ mergeInto(LibraryManager.library, {
           throw new FS.ErrnoError({{{ cDefine('EINVAL') }}});
         }
         var newsock = listensock.pending.shift();
+        if (!newsock) {
```

Comment:
How about combining this with the above condition to save a few bytes on code size?  e.g. `if (!listensock.server || !listensock.pending)`

Reply:
``!listensock.pending`` does not work, but ``!listensock.pending.length`` does.

Reply:
SGTM